python/pyspark/ml/fpm.py

```diff
@@ -23,17 +23,17 @@
 __all__ = ["FPGrowth", "FPGrowthModel"]
 
 
-class HasSupport(Params):
+class HasMinSupport(Params):
     """
-    Mixin for param support.
+    Mixin for param minSupport.
     """
 
     minSupport = Param(
         Params._dummy(),
         "minSupport",
-        """Minimal support level of the frequent pattern. [0.0, 1.0].
-        Any pattern that appears more than (minSupport * size-of-the-dataset)
-        times will be output""",
+        "Minimal support level of the frequent pattern. [0.0, 1.0]. " +
+        "Any pattern that appears more than (minSupport * size-of-the-dataset) " +
+        "times will be output in the frequent itemsets.",
```

Comment:
Note there will be ```\n``` in the generated Python API docs if using ```"""```.

Before this PR:
![image](https://cloud.githubusercontent.com/assets/1962026/26410581/501e99d4-40d6-11e7-86e7-c23ebda9b81e.png)

After this PR:
![image](https://cloud.githubusercontent.com/assets/1962026/26410589/583eed1c-40d6-11e7-8779-6cb8232756a6.png)